akka-actor-tests/src/test/scala/akka/actor/ActorDSLSpec.scala

```diff
@@ -16,7 +16,6 @@ import akka.testkit.TimingTest
 
 class ActorDSLDummy {
   //#import
-  import akka.actor.ActorDSL._
```

Comment:
This one is for the docs, keep it

Reply:
would it be possible to remove the top level ` import akka.actor.ActorDSL._` instead, so that tooling see this as necessary? and add it at more places inline where needed. Alternatively move the `//#import` to the top level.

Reply:
> would it be possible to remove the top level import akka.actor.ActorDSL._ instead, so that tooling see this as necessary?

I tried this locally. scalafix will fail to analyze. I will try to find more detail later.